src/Microsoft.OData.Client/DataServiceContext.cs

```diff
@@ -476,6 +476,13 @@ public virtual bool ApplyingChanges
             internal set { this.applyingChanges = value; }
         }
 
+
+        /// <summary>Gets or sets whether query projection will handle null propagation automatically. If set to true null propagation checks can be omitted from queries.</summary>

```

Comment:
```suggestion
        /// <summary>
        /// Gets or sets whether query projection will handle null propagation automatically. If set to true null propagation checks can be omitted from queries.
        ///</summary>
```

Reply:
Is this statement correct?
`If set to true null propagation checks can be omitted from queries.`

Reply:
That is correct.